__tests__/components/Layout.test.js

```diff
@@ -94,7 +94,7 @@ describe('Layout with default text', () => {
         locale="en"
         meta={meta}
         display={display}
-        children={aaChildrenProps}
+        // children={aaChildrenProps}
```

Comment:
This is fine for the time being but we should really look at properly fixing the component instead of commenting this out

Reply:
Yes, and this test was not even being run.